assets/activation-scripts/wrapper

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+# shellcheck shell=bash
+
+# Trace levels supported by activation scripts:
+#   1. (-v) top-level activate script
+#   2. (-vv) language-specific profile scripts
+#   3. (-vvv) zsh `autoload -U compinit` (very verbose)
+export _flox_activate_tracelevel="${_FLOX_PKGDB_VERBOSITY:-0}"
+[ "$_flox_activate_tracelevel" -eq 0 ] || set -x
+
+# Ensure that $_flox_activate_tracer is defined as an executable.
+if [ -z "${FLOX_ACTIVATE_TRACE-}" ]; then
+  # If FLOX_ACTIVATE_TRACE is empty or not set, set _flox_activate_tracer to
+  # `true` which can always be invoked with any arguments without error.
+  export _flox_activate_tracer=true
+else
+  # If FLOX_ACTIVATE_TRACE is set but does not refer to an executable, then
+  # set _flox_activate_tracer to the default trace script.
+  if [ -x "${FLOX_ACTIVATE_TRACE:-}" ]; then
+    export _flox_activate_tracer="$FLOX_ACTIVATE_TRACE"
+  else
+    export _flox_activate_tracer="__OUT__/activate.d/trace"
+  fi
+fi
+"$_flox_activate_tracer" "${BASH_SOURCE[0]}" "$@" START
+
+_dirname="@coreutils@/bin/dirname"
+_getopt="@getopt@/bin/getopt"
+_readlink="@coreutils@/bin/readlink"
+
+set -euo pipefail
+
+# These all derive from the `flox-interpreter` package.
+# FIXME This is wrong; the profile.d scripts in particular should be
+#       sourced from the environment itself so that users can add pkgs
+#       which add additional scripts to the etc/profile.d directory.
+_activate_d="__OUT__/activate.d"
+_profile_d="__OUT__/etc/profile.d"
+# shellcheck source-path=SCRIPTDIR/activate.d
+source "${_activate_d}/source-profile-d.bash"
+
+# Top-level Flox environment activation script.
+
+# Parse command-line arguments.
+OPTIONS="c:e:"
+LONGOPTS="command:,env:"
+USAGE="Usage: $0 [-c \"<cmd> <args>\"] [(-e|--env) <env>]"
+
+PARSED=$("$_getopt" --options="$OPTIONS" --longoptions="$LONGOPTS" --name "$0" -- "$@")
+# shellcheck disable=SC2181
+if [[ $? -ne 0 ]]; then
+  echo "Failed to parse options."
+  exit 1
+fi
+
+# Use eval to remove quotes and replace them with spaces.
+eval set -- "$PARSED"
+
+# Set default values for options.
+FLOX_CMD=""
+while true; do
+  case "$1" in
+    -c | --command)
+      shift
+      if [ -z "${1:-}" ]; then
+        echo "Option -c requires an argument." >&2
+        echo "$USAGE" >&2
+        exit 1
+      fi
+      FLOX_CMD="$1"
+      shift
+      ;;
```

Comment:
We might want to remove the `-c` option as well, i dont think we use it and this is non user facing.
Then again we neither use `activate -c` i think but still support the flag, presumably for debugging purposes(?).